rpc/executables.go

```diff
@@ -0,0 +1,155 @@
+package rpc
+
+import (
+	"encoding/json"
+
+	hintRunnerZero "github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/zero"
+	"github.com/NethermindEth/cairo-vm-go/pkg/parsers/zero"
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/jsonrpc"
+	"github.com/NethermindEth/juno/utils"
+)
+
+type CasmEntryPoint struct {
```

Comment:
Maybe it's good idea to add a comment with a link to spec - similar to other rpc methods?